docs/custom_domains.rst

```diff
@@ -51,10 +51,11 @@ You can also host your documentation from your own domain.
 
       .. admonition:: Certificate Authority Authorization (CAA)
 
-         If your custom domain — either the subdomain you're using or the root domain — has configured CAA records,
+         If your custom domain has configured CAA records,
          please do not forget to include Cloudflare CAA entries to allow them to issue a certificate for your custom domain.
+         CAA records are typically configured at the root domain.
          See the `Cloudflare CAA FAQ`_ for details.
-         We need a record that looks like this: ``0 issue "digicert.com"`` in response to ``dig +short CAA <domain>``
+         We need a record that looks like this: ``0 issue "digicert.com"`` in response to ``dig +short CAA <rootdomain>``
```

Comment:
Does it worth to put an example here? Like, if your domain is `docs.example.org`, your root domain is `example.org`.

Reply:
I reworked this section and added an example.